app/Jobs/Emails/Registration/Attendees/SummitAttendeeExcerptEmail.php

```diff
@@ -0,0 +1,79 @@
+<?php namespace App\Jobs\Emails\Registration\Attendees;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use App\Jobs\Emails\AbstractEmailJob;
+use App\Jobs\Emails\IMailTemplatesConstants;
+use App\Services\utils\IEmailExcerptService;
+use Illuminate\Support\Facades\Log;
+use models\summit\Summit;
+/**
+ * Class SummitAttendeeExcerptEmail
+ * @package App\Jobs\Emails\Registration\Attendees
+ */
+class SummitAttendeeExcerptEmail extends AbstractEmailJob
```

Comment:
please create an abstract base clase for all excerpt emails jobs
all share the same logic